rails-questions.md

```diff
@@ -2,38 +2,44 @@
 
 1. What is SQL?
 
-
+structured quary language is a domain specific language which is the language in which some databases is written in.structured quary language 
 
 2. What the PostgreSQL query that would return all the content in a particular table?
 
-
+SELECT * FROM
 
 3. What is the command to create a new Rails application with a PostgreSQL database?
 
-
+$ rails new app_name -d postgresql -T
 
 4. What is the command to generate a Rails model for a meals table with columns named breakfast, lunch, and dinner?
 
+$ rails generate model Meals breakfast: string lunch:string diner:string
 
 
 5. What is a migration? Why would you use one?
-
+A migration is a tool used in rails for chaging the schema. you would use it to update your rails app.
 
 
 6. What is the command to generate a migration file?
 
+$ rails db:migrate 
 
-
-7. What is the naming convention for generating a Rails model? What is the naming convention for the table that is generated with this command?
+7. What is the naming convention for generating a Rails model? What is the naming convention for the table that is generated with this command? the naming conventions used for a rails model would Pascal casing and the naming convention for a rails table would be snake_case. 
 
 
 
 8. What is the schema file in a Rails application? Can you modify the schema directly? Why or why not?
-
+ the schema is what hold the current information in the database. You cant modify the schema directly you would need to use the rails console i believe. 
+ could i get some clarification on this? 
```

Comment:
Yes, the scheme can only be updated through migrations. The scheme is the blueprint for the database and Rails controls how the blueprint is expressed.